dango/types/src/auth.rs

```diff
@@ -55,13 +56,14 @@ pub struct Metadata {
 #[grug::derive(Serde)]
 pub struct PasskeySignature {
     pub authenticator_data: Binary,
-    pub origin: String,
-    pub cross_origin: bool,
+    pub client_data: Binary,
     pub sig: ByteArray<64>,
 }
 
-/// Passkey client data.
-#[grug::derive(Serde)]
+/// Passkey client data
+/// We use serde instead of grug because it does not support unknown fields.
```

Comment:
Let's just remove `deny_unknown_fields` from grug derive. I'm going to push a commit